### ISSUE TITLE
Widgets page and Customizer: The inserter should not use ARIA tabs semantics

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#61653]
### Description

In the Widgets page, the Inserter doesn't need tabs because it is only supposed to show the 'Blocks' panel.

https://github.com/WordPress/gutenberg/pull/61510 hid the entire panel header via CSS. The panel header contains the wrapper with `role="tablist"` that contains the 3 `role="tab"` panels. They're now hidden with `display: none` which makes them not perceivable by assistive technology. However, the panel with blocks still use a `role="tabpanel"`, which is not associated with any 'tab'. As such, the ARIA tabs pattern is semantically broken.

Instead of using CSS, the inserter in the Widgets page should render a dirrerent UI that doesn't use the ARIA tabs pattern in the first palce, as it was before the latest changes to reunify the editor.

Cc @youknowriad @MaggieCabrera @scruffian 

### Step-by-step reproduction instructions

- Activate Twenty Twenty-One.
- Go to Appearance > Widgets.
- Open the main inserter.
- Observe the main inserter doesn't have tabs (they are hidden via CSS together with the whole panel header).
- Inspect the source via your browser dev tools.
- Obaserve the panel element with class `block-editor-inserter__tabpanel` still has a `role="tabpanel"` attribute.

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by afercia** [comment]
I updated this issue title to include the Customizer.

After latest changes, the 'inserter' panel in Customize > Widgets shows 3 tabs, including Patterns and Media. It is expected to only show the Blocks panel.

Also, after https://github.com/WordPress/gutenberg/pull/61421 the panel now has two 'Close' buttons. Screenshot:

On latest trunk:

![trunk](https://github.com/WordPress/gutenberg/assets/1682452/70b011fe-6048-468f-a041-eb31ad2684da)

On WordPress 6.5:

![wp 6 5](https://github.com/WordPress/gutenberg/assets/1682452/b27e4dce-34b6-47de-b6ea-816006bf8fe6)

As mentioned above, the unnecessary tabs can't be hidden with CSS. Instead, the panel should render a UI that doesn't use ARIA tabs.

**Comment by Rishit30G** [comment]
Thanks for sharing the issue, 

> Also, after https://github.com/WordPress/gutenberg/pull/61421 the panel now has two 'Close' buttons. 

I could also identify that the second cross button doesn't work, sharing a screencast for the same: 

https://github.com/user-attachments/assets/7e18fd9e-d9ac-4dbb-aebb-010a8e0185d5

Would it be a good idea to remove it from here, will need to check it's dependency 🤔 

https://github.com/WordPress/gutenberg/blob/c30b8c233fe90937b45c0e36ad6a213ed185878d/packages/block-editor/src/components/tabbed-sidebar/index.js#L69C1-L75C8

**Comment by sarthaknagoshe2002** [comment]
I was not able to reproduce this issue any more.
can someone could also get this re-checked.

**Comment by Infinite-Null** [comment]
Hi @sarthaknagoshe2002 I was able to reproduce the issue:

https://github.com/user-attachments/assets/e044d013-f3fd-4d3b-a4bb-e958f1cd1561

**Comment by Infinite-Null** [comment]
I further invetigated and found that here we are passing tabs manually not as prop: 
```
tabs={ [
						{
							name: 'blocks',
							title: __( 'Blocks' ),
							panel: (
								<>
									{ inserterSearch }
									{ selectedTab === 'blocks' &&
										! delayedFilterValue &&
										blocksTab }
								</>
							),
						},
						{
							name: 'patterns',
							title: __( 'Patterns' ),
							panel: (
								<>
									{ inserterSearch }
									{ selectedTab === 'patterns' &&
										! delayedFilterValue &&
										patternsTab }
								</>
							),
						},
						{
							name: 'media',
							title: __( 'Media' ),
							panel: (
								<>
									{ inserterSearch }
									{ mediaTab }
								</>
							),
						},
					] }
```

And in the widget, we don't need the pattern and media. Also, `TabbedSidebar` has its close button.

**Comment by sarthaknagoshe2002** [comment]
@Infinite-Null apologies for missing out on giving context,
I was referring to the close button's issue.

**Comment by Infinite-Null** [comment]
My Bad, I gave the wrong recording! I have updated the above comment!
